src/components/SpecialResolution/HelpSpecialResolution.vue

```diff
@@ -0,0 +1,200 @@
+<template>
+  <div>
+    <div class="help-section-component ma-6">
```

Comment:
If there's only ever 1 instance of this component on the page, please use an id instead of a classname.

Please change name to match the component's filename (help-special-resolution).